.github/workflows/helper-deploy-func.yaml

```diff
@@ -0,0 +1,115 @@
+name: Deploy Cloud Function
+
+on:
+  workflow_call:
+    inputs:
+      func-name:
+        required: true
+        type: string
+      func-src-dir:
+        required: true
+        type: string
+        description: Path in the repo containing this function's code
+      func-entrypoint:
+        required: true
+        type: string
+        description: Name of the function entrypoint in main.py
+      func-data-gcs-url:
+        required: true
+        type: string
+        description: GCS URL to the archive containing the data for this function
+      func-memory-mb:
+        required: true
+        type: number
+
+      func-runtime:
+        default: python311
+        type: string
+      func-svc-acct:
+        default: logging-monitoring@gap-som-dbmi-geneplx-app-p0n.iam.gserviceaccount.com
+        type: string
+      project-id:
+        default: gap-som-dbmi-geneplx-app-p0n
+        type: string
+      region:
+        default: us-central1
+        type: string
+      func-data-local-path:
+        default: 'data'
+        type: string
+        description: 'Path under func-src-dir where the GCS archive is extracted, default ./data'
+    secrets:
+      JSON_GCLOUD_SERVICE_ACCOUNT_JSON:
+        required: true
+jobs:
+  deploy-cloud-func-helper:
+    runs-on: 'ubuntu-latest'
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+
+    env:
+      DATA_ARCHIVE_HASH: 'none'
+
+    steps:
+      - uses: 'actions/checkout@v4'
+
+      - id: 'auth'
+        uses: 'google-github-actions/auth@v2'
+        with:
+          # workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
+          # service_account: 'cloud-function-deployer@gap-som-dbmi-geneplx-app-p0n.iam.gserviceaccount.com'
+          credentials_json: ${{ secrets.JSON_GCLOUD_SERVICE_ACCOUNT_JSON }}
+
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v2'
+        with:
+          version: '>= 363.0.0'
+
+      - name: Get hash of the data archive for this function
+        id: get-data-hash
```

Comment:
I think this is vestigial.

Reply:
I just meant the `id`, not the whole step. I don't think you're using the `id` anywhere. Completely inconsequential though.